.gitignore

```diff
@@ -119,7 +119,10 @@ secrets.json
 .env*
 
 # Ingnore build target
-contracts/target/
+protocol/target/ink/*/release
+protocol/target/ink/*/wasm32-unknown-unknown
```

Comment:
yes it is, that's the target we're aiming at. Ink always aims at wasm32-unknown-unknown because that's the format expected for contract code on the blockchain. I doubt they'll change it anytime soon. Can potentially see it becoming wasm64 one day, but certainly not soon